src/field/pokemon.ts

```diff
@@ -5427,6 +5429,16 @@ export default abstract class Pokemon extends Phaser.GameObjects.Container {
     );
 
     if (cancelled.value) {
+      return this.messageIsImmune(quiet);
```

Comment:
```suggestion
      return false;
```
We should trust the abilities to queue their own messages, that's a standard part of their activation. (this may have not worked before, but it will now because `quiet` is properly being passed as false)